net.sourceforge.Artha.yaml

```diff
@@ -0,0 +1,140 @@
+app-id: net.sourceforge.Artha
```

Comment:
since the website is `artha.sourceforge.net` is should be `net.sourceforge.artha.Artha`

Reply:
should I make a new pull request with the new name? Because, we need to rename direcortory, xml, yaml files, and every instance of the name.

Reply:
why a new pull request? the branch name is completely irrelevant, and that the only github doesn't like changing.

a new PR mean more work for volunteers

Reply:
@hfiguiere Should I rename the yaml and xml files too? or just change `app-id` from `net.sourceforge.Artha` to `net.sourceforge.artha.Artha` in yaml and xml files. I think it would be inconsistent to have different filename and app-id .

Reply:
filename MUST match

Reply:
changed file name and app-id as per your suggestion: https://github.com/flathub/flathub/pull/4670